AdfsEventsModule.psm1

```diff
@@ -1,8 +1,60 @@
 ﻿# Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+# ----------------------------------------------------
+#
+# Global Constants
+#
+# ----------------------------------------------------
 
-#Declare helper functions
+$global:CONST_ADFS_ADMIN = "AD FS"

```

Comment:
The $global variables effect the powershell session. I suggest using the $script scope to only add these variables to the local script.

Reply:
That makes sense, I'll make that change